api/time_bucket_ng.md

```diff
@@ -11,6 +11,17 @@ and could be removed in future releases. Use these features at your own risk, an
 do not use any experimental features in production.
 </highlight>
 
+Currently there are two main differences between `time_bucket()` and
+`time_bucket_ng()` which make them not 100% compatible:
+
+1. `time_bucket_ng()` doesn't work with timestamps prior to `origin`, while
+`time_bucket()` does. The reasons why `time_bucket_ng()` has this limitation
+are explained below;
+2. The default `origin` values differ. `time_bucket()` uses 3 Jan 2000, because
+it's Monday and it works better with weekly buckets. `time_bucket_ng()` uses
+1 Jan 2000 as a first day of the month and the year, to make the function
+work with monthly/annual aggregates by default.
```

Comment:
```suggestion
`time_bucket_ng()` doesn't work with timestamps prior to `origin`, while
`time_bucket()` does. This limitation is because the default `origin` values differ.
`time_bucket()` uses an origin date of 3 Jan 2000, because that date is a Monday,
which works better with weekly buckets. `time_bucket_ng()` uses an origin date of
1 Jan 2000, because it is the first day of the month and the year, which works better
with monthly or annual aggregates.
```

Reply:
This whole section should possibly also be wrapped in a `<highlight type="warning">`